test/access_control.sh

```diff
@@ -3,26 +3,20 @@
 # test access control
 source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
 source "${OS_O_A_L_DIR}/hack/testing/util.sh"
+current_dir=$(dirname "${BASH_SOURCE[0]}" )
+repo_dir=${repodir:-"$current_dir/../"}
+test_name=$(basename $0)
```

Comment:
It would enable re-usability and improve maintenance if `current_dir` moves to the the top. In addition keeping to `${BASH_SOURCE[0]}` instead of flipping with `$0` makes the script more robust when sourced.